Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -81,6 +81,14 @@ bash scripts/install_prerequisites.sh
 
    The `git remote -v` command at the end shows all your current remotes.
 
+   For developers that are using SSH to push to their git repository, please change the SSH config at `~/.ssh/config` to ensure that the git pre-push hook doesn't timeout at 5 minutes. In order to do this, add the following lines to `~/.ssh/config` ([Reference](https://stackoverflow.com/a/65818657)):
```

Comment:
For developers who are ...

timeout --> time out (since it's being used as a verb)

Ditto in two other places below.

Reply:
Done.